google-cloud-dataproc/google-cloud-dataproc.gemspec

```diff
@@ -0,0 +1,30 @@
+# -*- ruby -*-
+# encoding: utf-8
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-dataproc"
+  gem.version       = "0.1.0"
+
+  gem.authors       = ["Google LLC"]
+  gem.email         = "googleapis-packages@google.com"
+  gem.description   = "google-cloud-dataproc is the official library for Google Cloud Dataproc API."
+  gem.summary       = "API Client library for Google Cloud Dataproc API"
+  gem.homepage      = "https://github.com/googleapis/googleapis"
+  gem.license       = "Apache-2.0"
+
+  gem.platform      = Gem::Platform::RUBY
+
+  gem.files         = `git ls-files -- lib/*`.split("\n") +
+                      ["README.md", "LICENSE", ".yardopts"]
+  gem.require_paths = ["lib"]
+
+  gem.required_ruby_version = ">= 2.0.0"
+
+  gem.add_dependency "google-gax", "~> 1.0.0"
+  gem.add_dependency "googleapis-common-protos", "~> 1.3.1"
```

Comment:
This is actually something that needs to be fixed in the generator, but could you hand-edit to remove lines 24-25? The toolkit issue is https://github.com/googleapis/toolkit/issues/1752.

Reply:
Done.